### ISSUE TITLE
TCGA Disease Free Survival Issue

### ISSUE THREAD

**Issue by ecerami** [cBioPortal/cbioportal#3101]
Reported by end user:

I’m trying to reproduce your disease free survival plots but am unclear how your pipelines select from multiple relapse events for the same patients.  For example patient TCGA-CU-A72E has two new tumor event times: 256 days and 364 days.  Cbioportal selects 364 days.  Isn’t 256 more correct if we are trying to measure disease free survival?

More info from user:  

Here is a link to gdc data portal from which you can download clinical metadata for TCGA-CU-A72E: https://portal.gdc.cancer.gov/files/4f3ae24f-eecd-4ba3-a592-cb9af99ed6e2

Download, de-compress, open and go to follow-ups section.  Under follow-ups you’ll find two new tumor event entries (256 and 364). 

BLCA GDAC firehose merged clinical files also shows these two events.

Notes from Ethan:  Reached out to Ben, who confirmed that this calculation is done within one of the MSK pipelines (that is not currently on github).  Also confirmed with @schultzn that we should be using the first event, not the second.

**Comment by n1zea144** [comment]
I've done some digging into this - for starters, our pipeline uses the biotab (tab delimited) version of the clinical data, not the XML flavor.  Having said that, I've discovered that the XML file linking off of the GDC website contains different data than what is in the biotab formatted data (which now lives in the GDC legacy archive.  I've added a few links here for reference:

[TCGA BLCA files on GDC legacy archive](https://portal.gdc.cancer.gov/legacy-archive/files/8e327e50-f880-4ac6-b9a0-0e2b1c5b9dec).

The value of DFS_MONTHS that we computed (11.96) is coming from the new_tumor_event_dx_days_to (364) from this file:

https://portal.gdc.cancer.gov/legacy-archive/files/1f9b1ca1-56c0-4278-b725-e029adb08288

I've noticed that the XML file contained in the legacy archive is different from the XML file referenced above which leads me to believe that an update may have occurred which was not placed into the legacy archive (although the versions and upload dates of both versions are the same):

https://portal.gdc.cancer.gov/legacy-archive/files/8e327e50-f880-4ac6-b9a0-0e2b1c5b9dec

Assuming the XML files on GDC are the latest and greatest, we would have to prioritize rewriting our TCGA pipeline to convert XML versions of files and then reprocess all of our TCGA clinical data...no easy task.
